dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -67,7 +67,6 @@ com.sun.xml.internal.messaging.saaj.util.transform=com.sun.xml.messaging.saaj.ut
 io.grpc=io.grpc
 io.grpc.netty=io.grpc.netty
 io.openliberty.cdi.spi=io.openliberty.cdi.spi
-jakarta.el=jakarta.el
```

Comment:
Can you just explain to me why this is not necessary any longer?

Reply:
In the EL 5.0 PR, I needed to transform the "io.openliberty.jakarta.jstl.2.0" bundle in order to get features which depended on EL to resolve. 

By transforming o.openliberty.jakarta.jstl.2.0 (an EE9 bundle),  I needed the jakarta.el imports to update. However, the current transformation rules only applied to javax.el -> jakarta.el.  By stating jakarta.el=jakarta.el, the versions then updated. 

We have a new jstl 3.0 api, so this is rule is no longer necessary.